frontend/src/components/user-management/student/AddClassroomModal.tsx

```diff
@@ -18,21 +19,27 @@ import {
   VStack,
 } from "@chakra-ui/react";
 
+import CREATE_CLASS from "../../../APIClients/mutations/ClassMutations";
+import { ClassResponse } from "../../../APIClients/types/ClassClientTypes";
 import { Grade } from "../../../APIClients/types/UserClientTypes";
 import { PlusOutlineIcon } from "../../../assets/icons";
 import gradeOptions from "../../../constants/CreateAssessmentConstants";
+import AuthContext from "../../../contexts/AuthContext";
 import { ClassroomForm, ClassroomInput } from "../../../types/ClassroomTypes";
 import ErrorToast from "../../common/ErrorToast";
 import ModalFooterButtons from "../../common/ModalFooterButtons";
+import Toast from "../../common/Toast";
 
 import SelectFormInputClassroom from "./SelectFormInputClassroom";
 
 const AddClassroomModal = (): React.ReactElement => {
   const {
+    handleSubmit,
     watch,
     setValue,
     formState: { errors },
   } = useFormContext<ClassroomForm>();
+  const { authenticatedUser } = useContext(AuthContext);
   const { onOpen, onClose, isOpen } = useDisclosure();
   const [classNameError, setClassNameError] = React.useState(false);
```

Comment:
can we remove these and the associated lines (61-74, 79, 84, 89)? i don't think we actually need them